### PR TITLE
🌍 Globally importing _variables.scss so its available everywhere.

### PR DESCRIPTION
**Pull request by PixsaOJ**
Color variables can be used in almost all components or views, so making it available globally.

### CODE REVIEW

**Comment by w3cj** [comment]
Thanks for doing this! I'll take a look / merge the changes next time I stream.

File: client/src/components/Button.vue [hunk 1]
```diff
@@ -10,8 +10,6 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@import '../styles/abstracts/variables';
```
Comment (w3cj) [inline client/src/components/Button.vue:13]: wat????

File: client/src/components/ChatBox.vue [hunk 2]
```diff
@@ -38,8 +38,6 @@ export default {
 </script>
 
 <style lang="scss">
-@import '../styles/abstracts/variables';
-
```
Comment (w3cj) [inline client/src/components/ChatBox.vue:42]: who????

File: client/src/views/Messages.vue [hunk 3]
```diff
@@ -67,8 +67,6 @@ export default {
 </script>
 
 <style lang="scss">
-@import '../styles/abstracts/variables';
```
Comment (w3cj) [inline client/src/views/Messages.vue:70]: OKAY

**Review by w3cj — APPROVED** [review]
IGNORE ALL COMMENTS.

File: client/src/components/Button.vue (hunk 1 above)
Comment (PixsaOJ) [reply client/src/components/Button.vue:13]: : (

**Pull request merged**